### PR TITLE
Add zh-CN translation for i18n

### PR DESCRIPTION
**Pull request by doabit**

### CODE REVIEW

**Comment by graysonwright** [comment]
谢谢! 谁要确认这是正确？我的中文没有那么好。

**Comment by doabit** [comment]
@graysonwright 不客气，这个还是很简单的中文，没有什么问题。

File: config/locales/administrate.zh-CN.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+zh-CN:
+  administrate:
+    actions:
+      confirm: 确定?
```
Comment (graysonwright) [inline config/locales/administrate.zh-CN.yml]: I would expect to see something like `真的？` - thoughts?

Comment (doabit) [inline config/locales/administrate.zh-CN.yml]: Chinese is my mother language, In chinese, it should `确定`.

Comment (graysonwright) [inline config/locales/administrate.zh-CN.yml]: ok, sounds good. thanks!

**Comment by graysonwright** [comment]
Merged in as b19c478

**Pull request closed without merging**